proto/expr.proto

```diff
@@ -191,6 +191,7 @@ message ExprNode {
     // Internal functions
     VNODE = 1101;
     // Non-deterministic functions
+    NOW = 2022;
```

Comment:
Adding `NOW` as a variant of `ExprType` can be confusing, as we're sure that `ExprType` is actually `FunctionCallType` and `Now` is not a function call here...